signwriting_transcription/pose_to_signwriting/bin.py

```diff
@@ -66,12 +68,22 @@ def main():
 
     print('Preprocessing pose.....')
     temp_pose_path = temp_dir / 'pose.pose'
-    preprocess_single_file(args.pose, temp_pose_path, normalization=False)
+    preprocess_single_file(args.pose, temp_pose_path, normalization=True)
 
     print('Predicting signs...')
     temp_files = []
+    start = 0
     for index, segment in tqdm(enumerate(sign_annotations)):
-        np_pose = pose_to_matrix(temp_pose_path, segment[0], segment[1]).filled(fill_value=0)
+        end = sign_annotations[index + 1][0] if index + 1 < len(sign_annotations) else None
+        if args.strategies == 'wide':
+            end = (end + segment[1]) // 2 if index + 1 < len(sign_annotations) else None
+            np_pose = pose_to_matrix(temp_pose_path, start, end).filled(fill_value=0)
+            start = end
+        else:
+            end = end if end is not None else segment[1] + 300
+            np_pose = pose_to_matrix(temp_pose_path, segment[0] - (segment[0] - start) * 0.25
```

Comment:
you are loading the pose from disk as many times as there are segments. maybe `pose_to_matrix` should accept a pose object as well as a path?

Reply:
i fixed that